android/app/src/main/cpp/app/organicmaps/Framework.cpp

```diff
@@ -108,11 +108,11 @@ enum MultiTouchAction
   MULTITOUCH_CANCEL =  0x00000004
 };
 
-Framework::Framework()
-  : m_lastCompass(0.0)
-  , m_isSurfaceDestroyed(false)
-  , m_isChoosePositionMode(false)
+Framework::Framework(std::function<void()> && callback)
```

Comment:
nit: use a more descriptive name than a generic `callback`. Callback for which events?